ext/rinku/autolink.c

```diff
@@ -136,19 +139,19 @@ static size_t
 autolink_delim_iter(uint8_t *data, size_t link_end, size_t offset, size_t size)
 {
 	size_t next_link_end;
-        int iterations = 0;
+	int iterations = 0;
 	link_end = autolink_delim(data, link_end, offset, size);
 
-        while(link_end != 0) {
-          next_link_end = autolink_delim(data, link_end, offset, size);
-          if (next_link_end == link_end || iterations > 5) {
-            break;
-          }
-          link_end = next_link_end;
-          iterations++;
-        }
+	while(link_end != 0) {
+		next_link_end = autolink_delim(data, link_end, offset, size);
```

Comment:
that's just the way things go sometimes.  Also, the file is riddled with tabs, while I'm betting @shajith had his editor set to use spaces... this brings it back into sync with the original style.  You can thank @barker1977